### DISCUSSION TITLE
Seq suddenly stop even seq service is running

### DISCUSSION THREAD

**Discussion by dvt49** [datalust/seq-tickets#1277] [Help]
Hello,

My seq application was suddenly stop working for several times recently 
It working again after I restart the seq service. 
Here is the error message I found on the log.

Can someone help?

{"@t":"2021-08-05T10:53:27.0951944Z","@mt":"Error serving {RequestUrl} (token: {ErrorToken})","@l":"Error","@x":"Microsoft.Isam.Esent.Interop.EsentTermInProgressException: Termination in progress\r\n   at Microsoft.Isam.Esent.Interop.Api.JetEndSession(JET_SESID sesid, EndSessionGrbit grbit)\r\n   at Microsoft.Isam.Esent.Interop.Session.ReleaseResource()\r\n   at Microsoft.Isam.Esent.Interop.EsentResource.Dispose(Boolean isDisposing)\r\n   at Flare.Storage.Esent.Data.EsentDBSession.Dispose()\r\n   at Flare.Storage.Esent.EsentStorageExtent.<Get>d__17.System.IDisposable.Dispose()\r\n   at Flare.Storage.Esent.EsentStorageExtent.<Get>d__17.MoveNext()\r\n   at Flare.Events.EventStore.<Get>d__13.MoveNext()\r\n   at System.Linq.Enumerable.<TakeIterator>d__25`1.MoveNext()\r\n   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)\r\n   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)\r\n   at Flare.Queries.DataStore.GetFilter(Int32 count, Nullable`1 startAfter, Nullable`1 startFrom, Nullable`1 rangeStartUtc, Nullable`1 rangeEndUtc, String filter, Nullable`1 shortCircuitAfter, EventRetrievalStatistics& statistics)\r\n   at Seq.Server.Web.Api.EventsModule.QueryEvents(String combinedFilter, EventRetrievalStatistics& stats)\r\n   at Seq.Server.Web.Api.EventsModule.InSignal(Signal signal, Boolean excludeStats)\r\n   at Nancy.Routing.Route.<>c__DisplayClass4.<Wrap>b__3(Object parameters, CancellationToken context)","RequestUrl":"https://seq.abcxxx.net/api/events/signal?signal=signal-110%2Csignal-17&filter=%22test44%22&count=9&afterId=event-e876e4e351c208d9ca30374500000000&shortCircuitAfter=5000","ErrorToken":"6347ad805327475ca29a662ec701312c","RequestId":"4f8822c4-e422-4ad3-9fa5-fabd712640e7"}

**Comment by nblumhardt** [comment]
Hi! Thanks for getting in touch.

It looks like you're running a rather old (4.x?) Seq version. The ESENT component that's causing this error is no longer used by the event store in newer Seq builds, so upgrading (ideally to 2021.2.x) should prevent this happening in future.

Hope this helps,
Nick
